Common/cpp/Tools/ReanimatedVersion.cpp

```diff
@@ -68,11 +76,16 @@ void checkJSVersion(
         "See `https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooting#mismatch-between-c-code-version-and-javascript-code-version` for more details.");
     return;
   }
+}
+#else // NDEBUG
+void checkJSVersion(
+    jsi::Runtime &rnRuntime,
+    const std::shared_ptr<JSLogger> &jsLogger) {
+  // NOOP
```

Comment:
It would be good to explain why this function is NOOP and why the next function always returns true (i.e. in which configuration exactly we skip the check).